tsl/test/shared/expected/dist_queries.out

```diff
@@ -0,0 +1,19 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
Why not add this to existing `dist_query.sql`?

Reply:
Having tests as shared tests is preferable to the non-shared variant as a shared test does not incur the database creation overhead a normal test would.